src/components/MyObservations/MyObservationsContainer.js

```diff
@@ -101,6 +101,16 @@ const MyObservationsContainer = ( ): Node => {
       user_id: currentUserId
     }
   } );
+  const myObsLoaded = useStore( state => state.myObsLoaded );
+  const setMyObsLoaded = useStore( state => state.setMyObsLoaded );
+
+  useEffect( () => {
```

Comment:
I'd love for us to avoid using `useEffect` as much as we can and be more declarative.